### DISCUSSION TITLE
Hi i need help to download

### DISCUSSION THREAD

**Discussion by yendri404** [Eloston/ungoogled-chromium#1560] [Q&A] [answered]
Hello, good afternoon everyone, see, I want to download ungogoled crhomium but I can't find where to download it, I'm very new to using github, so if you could tell me how to download it, I would appreciate it very much. thx por reading me

**Comment by networkException** [comment, marked as answer]
There's a download section in the projects readme: [here](https://github.com/Eloston/ungoogled-chromium#downloads)
